reference/orders.v2.oas2.yml

```
@@ -412,10 +435,73 @@ paths:
                           value: '180'
                         - id: 230
                           value: '192'
+              Shipping Consignment:
+                value:
+                  customer_id: 0
+                  consignments:
+                    shipping:
+                      - first_name: Jane,
+                        last_name: Doe,
+                        company: Acme Corp,
+                        street_1: 12 Main St,
+                        street_2: Unit 5,
+                        city: Austin,
+                        state: Texas,
+                        zip: 78751,
+                        country: United States,
+                        country_iso2: US,
+                        email: jane@doe.com,
+                        phone: 469-538-6697
+                        line_items:
+                          - name: BigCommerce Coffee Mug
+                            quantity: 1
+                            price_inc_tax: 50
+                            price_ex_tax: 45
+                  billing_address:
+                    first_name: Jane
+                    last_name: Doe
+                    street_1: 123 Main Street
+                    city: Austin
+                    state: Texas
+                    zip: '78751'
+                    country: United States
+                    country_iso2: US
+                    email: janedoe@example.com
+              Download Consignment:
+                value:
+                  customer_id: 11
+                  consignments:
+                    downloads:
+                      - line_items:
+                        product_id: 184
+                        quantity: 3
+                  billing_address:
+                    first_name: Jane
+                    last_name: Doe
+                    street_1: 123 Main Street
+                    city: Austin
+                    state: Texas
+                    zip: '78751'
+                    country: United States
+                    country_iso2: US
+                    email: janedoe@example.com
         required: true
       responses:
         '200':
           $ref: '#/components/responses/order_Resp'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/response_400'
+              examples:
+                Empty consignments:
+                  value:
+                    - status: 400
+                      message: The field 'consignments' in invalid.
```

```suggestion
                      message: The `consignments` field is invalid.
```
If this is the actual message returned for the `400`, disregard this